.travis/deploy.sh

```diff
@@ -15,20 +15,35 @@ if [[ "$TRAVIS_OS_NAME" == "linux" && ${BUILD_LINUX_WHEELS} -eq 1 ]]; then
     linux32 docker run --rm -e BUILD_GMP_CPU="i686" -v $(pwd):/io quay.io/pypa/manylinux1_i686 /io/.travis/build-linux-wheels.sh
     .travis/build_windows_wheels.sh
 else
-	if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-		python -m pip install wheel
-		python setup.py bdist_wheel
-	fi
-
-	if [[ "$TRAVIS_PYTHON_VERSION" == "pypy3" ]]; then
-		python -m pip install wheel
-		python setup.py bdist_wheel
-		python3 -m pip install wheel auditwheel pyelftools typing
-		wget -q https://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.bz2 && tar -xjpf patchelf-*.tar.bz2 && cd patchelf* && ./configure > /dev/null && sudo make install > /dev/null && cd ..
-		auditwheel repair dist/coincurve*.whl
-		rm dist/coincurve*.whl
-		mv wheelhouse/coincurve*.whl dist
-	fi
+    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        # Make sure we can build and "fix" the wheel.
+        python -m pip install delocate wheel
+        # Create directories for the built and fixed wheels.
+        mkdir dist_wheels/ fixed_wheels/
+        # Build the wheel for the local OS.
+        python -m pip wheel . --wheel-dir dist_wheels/
```

Comment:
@ofek I made the assumption that `deploy.sh` is run from the project root. If not, then `pip wheel .` will need to be changed to refer to a different path.

Also, you may be out of luck using `delocate` for PyPy on OS X. It might work, I'm not sure. I'm also not sure what `auditwheel` does on OS X. (**UPDATE**: I just realized that the `"$TRAVIS_PYTHON_VERSION" == "pypy3"` block is **not** happening within the `"$TRAVIS_OS_NAME" == "osx"` block. Oops.)

Reply:
Thanks for the heads-up! Yes I haven't had time yet to figure out how to build for PyPy3 on macOS.